Class 'Symplify\BetterReflectionDocBlock\CleanDocBlockFactory' not found

Hi, running `vendor/bin/rector process src --level nette24` i got following error:

```
[ERROR] Class 'Symplify\BetterReflectionDocBlock\CleanDocBlockFactory' not found in                                    
         /private/var/folders/2m/28z_py912fqff7x8sr0x0rxw0000gn/T/_rector_cache/ContainerCjlV94i/getDocBlockFactoryService.php on line 8   
```

It seems that there is `symplify/better-reflection-docblock` dependency missing in rector composer dependencies.

--- a/composer.json
+++ b/composer.json
@@ -19,6 +19,7 @@
         "symfony/console": "^4.0",
         "symfony/dependency-injection": "^4.0",
         "symfony/finder": "^4.0",
+        "symplify/better-reflection-docblock": "^3.2",
         "symplify/package-builder": "^3.2"
     },
     "require-dev": {

